scripts/mkcloud

```diff
@@ -56,6 +56,17 @@ mkcconf=mkcloud.config
 rm -f $mkcconf # we dont want to source old info in the line below
 source $qa_crowbarsetup
 
+: ${admin_image_password:='linux'}
+: ${architectures:='aarch64 x86_64 s390x'}
+: ${clouddata_nfs:=$clouddata}
+: ${clouddata_nfs_dir:='srv/nfs'}
+: ${repopath_sles12sp1:='repos/$arch/SLES12-SP1-Pool'}
+: ${repopath_sles12sp1up:='repos/$arch/SLES12-SP1-Updates'}
+: ${manila_service_image_path:='images/other'}
```

Comment:
FYI: we are currently refactoring the repositories and installation sources handling. Our goal is to make it easier and with less variables. So I really recommend not to introduce something like this now - it just complicates the refactoring. Thanks.